examples/sample_measurement/SampleMeasurement.serviceconfig

```diff
@@ -4,7 +4,7 @@
       "displayName": "Sample Measurement (Py)",
       "serviceClass": "ni.examples.SampleMeasurement_Python",
       "descriptionUrl": "",
-      "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
+      "providedInterfaces": [ "ni.measurementlink.measurement.v2.MeasurementService" ],
```

Comment:
https://github.com/ni/measurementlink-python/pull/306 supersedes this. You may want to just take Brad's code from there. He's adding both interfaces.

Reply:
I merged latest and took Brad's changes for this file.